pageserver/src/tenant.rs

```diff
@@ -6338,87 +6301,38 @@ mod tests {
             .create_test_timeline(TIMELINE_ID, Lsn(0x10), DEFAULT_PG_VERSION, &ctx)
             .await?;
 
-        let cancel = CancellationToken::new();
-
         let mut base_key = Key::from_hex("000000000033333333444444445500000000").unwrap();
         let mut base_key_child = Key::from_hex("000000000033333333444444445500000001").unwrap();
         let mut base_key_nonexist = Key::from_hex("000000000033333333444444445500000002").unwrap();
         base_key.field1 = AUX_KEY_PREFIX;
         base_key_child.field1 = AUX_KEY_PREFIX;
         base_key_nonexist.field1 = AUX_KEY_PREFIX;
```

Comment:
I commented on some other PR about this, why not just put the prefix in the key?

Reply:
yep, I'm going to fix that...